default_filters/README.md

```diff
@@ -0,0 +1,9 @@
+# Saved Filters
```

Comment:
Not sure if we should have it under root or under resources...
I assume that eventually we'd put them into installer and .app